tools/ci_build/github/linux/build_linux_arm64_python_package.sh

```diff
@@ -7,13 +7,16 @@ CXXFLAGS="-Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fstack-protector-st
 
 BUILD_DEVICE="CPU"
```

Comment:
This script is only used in python packaging pipeline. You can remove the BUILD_DEVICE arg,which was added by you, if we don't need to have two pipelines that one for with AzureEP, one for without AzureEP.

Reply:
No - the device sits there long before AzureEP exists, it is to differentiate hardware distinctions between platforms such as X86 / ARM, hence the name.

Reply:
You are right.